ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableDataProvider.kt

```diff
@@ -24,7 +24,7 @@ internal class VariableDataProvider(
     }
 
     fun productName(rcPackage: Package): String {
-        return "PRODUCT_NAME"
+        return rcPackage.product.title
```

Comment:
Is this localized?

Reply:
The localization here comes from Google. Similar to iOS I think. So depending on the Google account country, Google will return us a different title. I don't think we can do much ourselves here.

Reply:
Yeah perfect 👍🏻  that's what I was wondering.